utils/deploy.ts

```diff
@@ -209,6 +212,7 @@ export const deployAssetContracts = async ({
           options: { initializer: 'initialize(address)' },
         })
       : undefined;
+  await lNoriInstance?.deployed();
```

Comment:
is this tenderly related or is this just something that should have been awaited in the first place?

Reply:
the latter